source/config.py

```diff
@@ -0,0 +1,21 @@
+
+# path to the local copies of the data
+local_path = "local/"
+
+# path to the remove data store
+remote_path = "remote/"
+
+# size of an experimental data array
+size = [1000,10]
```

Comment:
I dig this config file. Let's move it to a config folder and rename it dev (or test, or debug), just something to distinguish from a production type configuration. That way we can test with multiple different configuration files. Also, all these config should/could probably be all uppercase since they likely won't change inter-run

Reply:
Easily done and done.